tests/test_cliCmdRemoveRuns.py

```diff
@@ -83,8 +95,25 @@ def test_removeRuns(self):
             root = "repo1"
             MetricTestRepo(root, configFile=os.path.join(TESTDIR, "config/basic/butler.yaml"))
 
-            # Execute cmd but say no, check for expected outputs.
+            # add the run to a CHAINED collection
+            parentCollection = "parent"
+            result = self.runner.invoke(
+                butler.cli, f"collection-chain {root} {parentCollection} ingest/run".split()
+            )
+            self.assertEqual(result.exit_code, 0, clickResultMsg(result))
+
+            # Execute cmd, should fail because there is a parent CHAINED
+            # collection
             result = self.runner.invoke(butler.cli, ["remove-runs", root, "ingest*", "--no-confirm"])
+            self.assertNotEqual(result.exit_code, 0, clickResultMsg(result))
+            self.assertIn(
+                mustBeUnlinkedMsg.format(run="ingest/run", parents=f'"{parentCollection}"'), result.output
+            )
+
+            # Execute cmd with --force
+            result = self.runner.invoke(
+                butler.cli, ["remove-runs", root, "ingest*", "--no-confirm", "--force"]
+            )
```

Comment:
Can we also run `butler query-collections` for the run to show it has disappeared?